smart-contracts/assembly/contracts/FT/WMAS.ts

```diff
@@ -33,9 +46,20 @@ export function withdraw(bs: StaticArray<u8>): void {
   const recipient = new Address(
     args.nextString().expect('recipient is missing'),
   );
-
-  assert(amount > 0, 'Payment must be more than 0 WMAS');
-
   burn(u256ToBytes(u256.fromU64(amount)));
   transferCoins(recipient, amount);
 }
+
+function computeStorageCost(receiver: Address): u64 {
+  let cost = 0;
+  if (!Storage.hasOf(Context.callee(), balanceKey(receiver))) {
+    // baseCost = NEW_LEDGER_ENTRY_COST = STORAGE_BYTE_COST * 4 = 100_000 * 4 = 400_000
+    cost = 400_000;
```

Comment:
Your code should be more verbose, making fewer comments necessary. Here is my suggestion:
- Create a `STORAGE_COST_PER_BYTE` constant.
- Create a `NODE_STORAGE_PREFIX_SIZE_BYTES` constant.

Then your code becomes readable without any comments.
```suggestion
const baseCost = STORAGE_COST_PER_BYTE * NODE_STORAGE_PREFIX_SIZE_BYTES
```